examples/dp_sgd_linear_regression.py

```diff
@@ -30,6 +30,7 @@ def forward(self, x: Tensor) -> Tensor:
 test_dataloader = DataLoader(test_dataset, batch_size=2)
 
 with Connection("localhost", 50051, default_secret=b"secret") as client:
+    print(client)
```

Comment:
There shouldn't be a print here, except for debugging purposes

Reply:
Yeah, forgot.
will clear it.